### PR TITLE
LOG_FILEPATH introduced to not write into sources location

### PR DESCRIPTION
**Pull request by vi4m**
Could be redefined from: 
- docker 
- deb package
- generic installation

### CODE REVIEW

File: src/ralph/settings/base.py [hunk 1]
```diff
@@ -11,6 +11,7 @@ def os_env_true(var, default=''):
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
 SECRET_KEY = os.environ.get('SECRET_KEY', 'CHANGE_ME')
+LOG_FILEPATH = os.environ.get('LOG_FILEPATH', '/tmp/runtime.log')
```
Comment (mkurek) [inline src/ralph/settings/base.py:15]: What do you think about `/tmp/ralph.log`?

Comment (vi4m) [inline src/ralph/settings/base.py:15]: good idea.

**Comment by mkurek** [comment]
:ok:

**Comment by ghost** [comment]
:ok:

**Pull request merged**